fabtests/common/hmem_neuron.c

```diff
@@ -58,6 +58,7 @@ struct neuron_ops {
 
 static void *neuron_handle = NULL;
 static struct neuron_ops neuron_ops;
+static bool nrt_initialized = false;
```

Comment:
This static variable can be inside the function.

Reply:
Thank you! Fixed in the newest revision.